CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Hardening: Mongo driver now compiled using --use-sasl-client --ssl to enable proper DB authentication mechanisms
 - Fix: correct error payload using errorCode (previously orionError was used) in POST /v1/queryContext and POST /v1/updateContext in some cases
 - Hardening: modification of the URL parsing mechanism, making it more efficient, and the source code easier to follow (#3109, step 1)
+- Fix : Missing or empty metadata values were not allowed in NGSIv1 create/update operations (#3121)

```

Comment:
NGSIv1 -> NGSIv2

My fault... I told NGSIv1 firs time (it was a typo) :(

Reply:
fixed in 25fa92f